cypress/integration/submit_spec.js

```diff
@@ -59,6 +65,12 @@ describe('e2e test', () => {
         cy.get('#field-input--lastname').clear()
         cy.get('#field-input--firstname').click()
         cy.get('#field-error--lastname>span').should('contain', 'Please fill in your last name')
+        cy.get('#field-input--lastname').type('test@')
+        cy.get('#field-error--lastname>span').should('contain', 'This field only accepts alphanumeric characters and , . ( ) / & \' - ')
```

Comment:
same here there's double appostrophes and a back slash in the last name error message which is not there in the pattern.

Reply:
Not the problem. The problem was that I was not clearing the input field before entering a new value.

Reply:
The backslash is to escape the `'` which is enclosed in single quotes.